data/zilliqa/assets/zil1w5hwupgc9rxyuyd742g2c9annwahugrx80fw9h/meta.json

```diff
@@ -1,16 +1,20 @@
 {
   "name": "Gary",
   "symbol": "GARY",
-  "web": "http://thegarytoken.com",
+  "web": "https://thegarytoken.com",
   "supply": 100,
   "decimals": 4,
   "email": "thegarytoken@gmail.com",
+  "publicTeam" : true,
   "links": {
     "twitter": "https://twitter.com/GusSnail",
-    "telegram": "https://t.me/TheGARYToken"
+    "telegram": "https://t.me/TheGARYToken",
+    "github" : "https://github.com/darthgus",
```

Comment:
remove this github, no repo related to the project

Reply:
done